src/site/_includes/layouts/_base.njk

```diff
@@ -15,6 +15,7 @@
     {%- endif -%}
```

Comment:
Didn't really want to create more layout templates so made the base one more configurable.